src/News Articles/NewsArticles.module.scss

```diff
@@ -0,0 +1,85 @@
+@import '../generalStyle.scss';
```

Comment:
Could you try:
- deleting commented out code
- reducing the nesting
- using classes for everything rather than tags etc.

having lots of nesting makes it much harder when it comes to things like
- understanding what applies when
- adding media queries
- adding psuedo classes.

I'm sure all of this works just fine! :) 
Just think it will be a little easier to understand and maintain if we use nesting only when it has real meaning.

ie if we want to covey some a relationship then nesting is great 
- here is a thing... and when you hover over it this happens... nesting the hover make it really clear. 
- here is a container... and then things inside it have these margins - again the relationship between the container and item is important... so the nesting helps make that clear)

but if its just defining things on a page then its usually better to keep them separate.

Reply:
actually, with a little more though I think I've pinned down a clearer version of what I mean.

- if its styling that applies *because its inside that parent* then nesting makes that really clear.
- if its styling that would still apply if the parent didn't exist, then nesting probably isn't a good idea

Does that make sense?